Makefile

```diff
@@ -169,7 +169,7 @@ check-docs: $(EMBEDMD) $(LICHE) build
 .PHONY:shell-format
 shell-format: $(SHFMT)
 	@echo ">> formatting shell scripts"
-	@$(SHFMT) -i 2 -ci -w -s $(shell find . -type f -name "*.sh" -not -path "*vendor*" -not -path "tmp/*")
+	@$(SHFMT)shell -i 2 -ci -w -s $( find . -type f -name "*.sh" -not -path "*vendor*" -not -path "tmp/*")
```

Comment:
This doesn't seem right.
`make: /tmp/.bin/shfmt-v3.1.2shell: Command not found`

Reply:
yea this is totally wrong, just typo